pkg/lib/util/util.go

```diff
@@ -9,11 +9,12 @@ import (
 	liburl "net/url"
 	"strconv"
 
-	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
 	liberr "github.com/konveyor/forklift-controller/pkg/lib/error"
 	core "k8s.io/api/core/v1"
 )
 
+const Insecure = "insecureSkipVerify"
```

Comment:
same here, it would be nicer to place the settings we support in a single place - is there a reason not to import the api package still? seems you resolved the cyclic dependency, no?